pkgs/by-name/bi/bitbox/package.nix

```diff
@@ -0,0 +1,84 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  buildNpmPackage,
+  clang,
+  go,
+  libsForQt5,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "bitbox";
+  version = "4.45.0";
+
+  src = fetchFromGitHub {
+    owner = "BitBoxSwiss";
+    repo = "bitbox-wallet-app";
+    rev = "v${version}";
+    fetchSubmodules = true;
+    hash = "sha256-miy7uLNWvLacLOrlpsUE/nR/JNBkcueJX8/mX+abeAg=";
+  };
+
+  patches = [
+    ./genassets.patch
+  ];
+
+  postPatch = ''
+    substituteInPlace frontends/qt/resources/linux/usr/share/applications/bitbox.desktop \
+        --replace-fail 'Exec=BitBox %u' 'Exec=bitbox %u'
+  '';
```

Comment:
```suggestion
  postPatch = ''
    substituteInPlace frontends/qt/resources/linux/usr/share/applications/bitbox.desktop \
      --replace-fail 'Exec=BitBox %u' 'Exec=bitbox %u'
    substituteInPlace frontends/qt/genassets.sh \
      --replace-fail '/usr/bin/find' 'find'
    
    patchShebangs --build frontends/qt/genassets.sh
  '';
```

Reply:
[`genassets.sh`][1] has no shebang at all therefore it is left untouched by `patchShebangs`, resulting in:

```console
...
./genassets.sh
make: ./genassets.sh: No such file or directory
...
```

[1]: https://github.com/BitBoxSwiss/bitbox-wallet-app/blob/master/frontends/qt/genassets.sh